openedx/features/lums/models.py

```diff
@@ -0,0 +1,32 @@
+"""
+Models for LUMS custom Information.
+"""
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class UserProfileExtension(models.Model):
+    """
+    This model is used to store additional custom profile fields.
+    """
+
+    empty = 'punjab'
```

Comment:
nit: I think we should remove this `empty` variable

Reply:
Done